ddtrace/settings/config.py

```diff
@@ -743,8 +743,15 @@ def _handle_remoteconfig(self, data, test_tracer=None):
             log.warning("unexpected number of RC payloads %r", data)
             return
 
+        # Check if 'lib_config' is a key in the dictionary since other items can be sent in the payload
+        config = None
+        for config_item in data["config"]:
+            if isinstance(config_item, Dict):
+                if "lib_config" in config_item:
+                    config = config_item
+                    break
```

Comment:
nit you can join the ifs:
```suggestion
            if isinstance(config_item, Dict) and "lib_config" in config_item:
                config = config_item
                break
```